bindings/java/iota-wallet-java/README.md

```diff
@@ -21,7 +21,33 @@ In addition, make sure you have the latest Java Development Kit (JDK) installed.
 After you linked the library, you can create a Wallet instance and interface with it.
 
 ```java
-...
+import org.iota.Wallet;
+import org.iota.types.AccountHandle;
+import org.iota.types.ClientConfig;
+import org.iota.types.WalletConfig;
+import org.iota.types.exceptions.WalletException;
+import org.iota.types.secret.MnemonicSecretManager;
+
+public class CreateAccounts {
+    private static final String SHIMMER_TESTNET_NODE_URL = "https://api.testnet.shimmer.network";
+    private static final int SHIMMER_COIN_TYPE = 4219;
+    private static final String DEFAULT_DEVELOPMENT_MNEMONIC = "hidden enroll proud copper decide negative orient asset speed work dolphin atom unhappy game cannon scheme glow kid ring core name still twist actor";
+
+    public static void main(String[] args) throws WalletException {
+        // Build the wallet.
+        Wallet wallet = new Wallet(new WalletConfig()
+                .withClientOptions(new ClientConfig().withNodes(new String[] { SHIMMER_TESTNET_NODE_URL }))
+                .withSecretManager(new MnemonicSecretManager(DEFAULT_DEVELOPMENT_MNEMONIC))
```

Comment:
Can you do the Stronghold version here?
Also needs storeMnemonic() then below, but since we recommend Stronghold over the MnemonicSecretManager, we should also have it in the example

Reply:
Updated the examples - should be fine like this?